packages/components/src/popover/README.md

```diff
@@ -151,15 +151,26 @@ Set this to hide the arrow which visually indicates what the popover is anchored
 -   Required: No
 -   Default: `true`
 
+### anchor
+
+The element that should be used by the `Popover` as its anchor. It can either be an `Element` or, alternatively, a `VirtualElement` — ie. an object with the `getBoundingClientRect` function and the `ownerDocument` properties defined.
+
+-   Type: `Element | VirtualElement`
+-   Required: No
+
 ### anchorRect
 
+**Deprecated**. Please use the `anchor` prop instead.
+
 A custom `DOMRect` object at which to position the popover. `anchorRect` is used when the position (custom `DOMRect` object) of the popover needs to be fixed at one location all the time.
 
 -   Type: `DOMRect`
 -   Required: No
 
 ### getAnchorRect
 
+**Deprecated**. Please use the `anchor` prop instead.
```

Comment:
Should we mention the version that these props will be removed?

Reply:
Good point, I'll update this at a later point.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/43691/commits/382b8699f909a2ea1134ac15c42284dab17fc420